src/explorer.ts

```diff
@@ -91,4 +111,39 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
     constructor(kind: kuberesources.ResourceKind, readonly id: string) {
         this.resourceId = kind.abbreviation + '/' + id;
     }
-}
\ No newline at end of file
+}
+
+class VirtualKubernetesResource implements KubernetesObject, ResourceNode {
+    readonly resourceId: string;
+
+    constructor (readonly id : string, readonly kind : string, readonly metadata?: any) {
+        this.resourceId = kind + "/" + id;
+    }
+
+    getChildren() : vscode.ProviderResult<KubernetesObject[]> {
+        if (this.kind === 'cluster') {
+            return [
+                new KubernetesKind(kuberesources.allKinds.node),
+                new VirtualKubernetesResource('Workloads', 'workload'),
```

Comment:
I don't quite get why deployments, jobs and pods are grouped under 'Workloads'.  Is it the 'making it more similar to the k8s dashboard' thing mentioned in the PR comments?  (If so, that's great and I agree we should do it - just me not understanding.)

Reply:
Right. It's same as the k8s dashboard ui. See https://github.com/kubernetes/dashboard